gma/src_java/com/google/firebase/gma/internal/cpp/AdInspectorHelper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.gma.internal.cpp;
+
+import android.util.Log;
+import com.google.android.gms.ads.AdError;
+import com.google.android.gms.ads.AdInspectorError;
+import com.google.android.gms.ads.MobileAds;
+import com.google.android.gms.ads.OnAdInspectorClosedListener;
+
+/**
+ * Helper class for listening to AdInspector closed events.
+ */
+public final class AdInspectorHelper implements OnAdInspectorClosedListener {
+  /**
+   * Pointer to the C++ AdInspectorClosedListener object to invoke
+   * when the AdInsepctor has been closed.
+   */
+  private long mNativeCallbackPtr;
```

Comment:
Do you need a method to zero this out, e.g. if the GMA instance is deleted?

Reply:
I don't believe that the instance CAN be deleted.